openassessment/data.py

```diff
@@ -96,43 +97,99 @@ def map_anonymized_ids_to_usernames(anonymized_ids):
+        .values("email", "anonymous_id")
+    )
+    anonymous_id_to_email_mapping = {
+        user["anonymous_id"]: user["email"] for user in users
+    }
+    return anonymous_id_to_email_mapping
+
+
+def map_anonymized_ids_to_fullname(anonymized_ids):
+    """
+    Args:
+        anonymized_ids - list of anonymized user ids.
+    Returns:
+        dictionary, that contains mapping between anonymized user ids and
+        actual user fullname.
+    """
+    User = get_user_model()
+
+    users = _use_read_replica(
+        User.objects.filter(anonymoususerid__anonymous_user_id__in=anonymized_ids)
+        .select_related("profile")
+        .annotate(anonymous_id=F("anonymoususerid__anonymous_user_id"))
+        .values("profile__name", "anonymous_id")
+    )
+
+    anonymous_id_to_fullname_mapping = {
+        user["anonymous_id"]: user["profile__name"] for user in users
+    }
+    return anonymous_id_to_fullname_mapping
+
+
 class CsvWriter:
     """
     Dump openassessment data to CSV files.
     """
 
     MODELS = [
-        'assessment', 'assessment_part',
-        'assessment_feedback', 'assessment_feedback_option',
-        'submission', 'score'
+        "assessment",
+        "assessment_part",
+        "assessment_feedback",
+        "assessment_feedback_option",
+        "submission",
+        "score",
     ]
 
     HEADERS = {
-        'assessment': [
-            'id', 'submission_uuid', 'scored_at',
-            'scorer_id', 'score_type',
-            'points_possible', 'feedback',
-        ],
-        'assessment_part': [
-            'assessment_id', 'points_earned',
-            'criterion_name', 'criterion_label',
-            'option_name', 'option_label', 'feedback'
+        "assessment": [
+            "id",
+            "submission_uuid",
+            "scored_at",
+            "scorer_id",
+            "score_type",
+            "points_possible",
+            "feedback",
         ],
-        'assessment_feedback': [
-            'submission_uuid', 'feedback_text', 'options'
+        "assessment_part": [
+            "assessment_id",
+            "points_earned",
+            "criterion_name",
+            "criterion_label",
+            "option_name",
+            "option_label",
+            "feedback",
         ],
-        'assessment_feedback_option': [
-            'id', 'text'
+        "assessment_feedback": ["submission_uuid", "feedback_text", "options"],
+        "assessment_feedback_option": ["id", "text"],
+        "submission": [
+            "uuid",
+            "student_id",
+            "item_id",
+            "submitted_at",
+            "created_at",
+            "raw_answer",
         ],
-        'submission': [
-            'uuid', 'student_id', 'item_id',
-            'submitted_at', 'created_at', 'raw_answer'
+        "score": [
+            "submission_uuid",
+            "points_earned",
+            "points_possible",
+            "created_at",
```

Comment:
Same comment here ^. Let's not add style changes to the lines we haven't changed